se/server/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>kumuluz-server-se</artifactId>

```

Comment:
I don't think an additional module is required. This module is dependent on kumuluzee-cdi-weld-se and kumuluzee-cdi-weld-se cannot be used in any other context that with this module. I suggest we move the contents of this module to kumuluzee-cdi-weld-se.